contrib/modelci/README.md

```diff
@@ -0,0 +1,149 @@
+# Model CI
+
+The users occasionally found a negligible performance or precision issue between different Paddle versions. Though we have unit tests for each class and Travis-CI to ensures the precision of each operator, there is no any logic to ensure the model (a composition of several operators) works as reliable as the operators.
+
+There are several conditions where an existing model will fail either in performance or precision:
+
+1. the incomplete coverage test cases, such as lacking the test of precision. 
+2. poor performance update, currently, we have no performance tracker for each operator.
+3. API changes, developers are likely to forget to update usages in some other repositories such as paddle/models.
```

Comment:
This sentence will be `Developers generally forget to update API use in other repositories such as paddle/models.`